example-get-started-experiments/code/.github/workflows/deploy-model-template.yml

```diff
@@ -0,0 +1,67 @@
+name: Test Model Artifact
+
+on:
+  # the workflow is triggered whenever a tag is pushed to the repository
+  push:
+    tags:
+      - "*"
+  # we can also run the workflow manually...
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Manual version name'
+        required: true
+        type: string
+  
+permissions:
+  contents: write
+  id-token: write
+
+jobs:
+
+  # This job parses the git tag with the GTO GitHub Action to identify model registry actions
+  parse:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: "Parse GTO tag"
+      id: gto
+      uses: iterative/gto-action@v2
+    outputs:
+      event: ${{ steps.gto.outputs.event }}
+      name: ${{ steps.gto.outputs.name }}
+      stage: ${{ steps.gto.outputs.stage }}
+      version: ${{ steps.gto.outputs.version }}
+
+  test-model-artifact:
+    needs: parse
+    # using the outputs from the "parse" job, we run this job only for actions
+    # in the model registry and only when the model was assigned to a stage called "test"
+    if: "${{ needs.parse.outputs.event == 'assignment' }} && ${{ needs.parse.outputs.stage == 'test' }}"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.8'
+        cache: 'pip'
+        cache-dependency-path: requirements.txt
+
+    - run: pip install -r requirements.txt
+
+    # this step adds our project's DVC remote to the workflow environment
+    - name: Add DVC Remote
+      run:  dvc remote add -d --local storage s3://dvc-public/remote/get-started-pools
```

Comment:
If we use `dvc artifacts get`, we should be able to drop all these steps as long as dvc is installed (which we should add as a step).